ush/eva/gen_eva_obs_yaml.py

```diff
@@ -94,5 +111,8 @@ def gen_eva_obs_yaml(inputyaml, templateyaml, outputdir):
     parser.add_argument('-i', '--inputjediyaml', type=str, help='Input JEDI YAML Configuration', required=True)
     parser.add_argument('-t', '--templateyaml', type=str, help='Template EVA YAML', required=True)
     parser.add_argument('-o', '--outputdir', type=str, help='Output directory for EVA YAMLs', required=True)
+    parser.add_argument('-g', '--group', type=str, help='ioda groups [ObsValue, ombg, ...] ', required=False, default='ObsValue')
+    parser.add_argument('-v', '--variable', type=str, help='Variable name ...', required=False, default='all')
+    parser.add_argument('-b', '--bound', type=str, nargs='+', help='min, max', required=False, default=[0, 20])
```

Comment:
@mjhossen ,  why are [0, 20] default bounds? That is not going to reproduce the figures that the other users were generating.

Reply:
@guillaumevernieres I put it based on obsvalue of sea-surface temperature. Can you please suggest that could be the appropriate for most? If someone needs, they can provide the bound. FYI, group and bound do not work unless there is an option in the templateyaml, like the one I create for marine. I guess templateyaml would be different for atm. But generic script works for both atm and marine.

Reply:
We really need the default to either be a) min,max of the data or b) have a series of defaults keyed off of variable name and group (i.e. OmB vs ObsValue)

Reply:
@CoryMartin-NOAA Thanks.

Reply:
It seems min/max would be more appropriate for automation. I might be wrong.

Reply:
the problem with min/max is then that 2 figures cannot be easily compared. I think that though is fine for default for this PR's purpose, we will likely need to add better defaults to EVA to handle realistic defaults to ensure consistency.

Reply:
@mjhossen , I have not looked at the code, but I suspect EVA was making use of default values set by the plotting routine ... Why not just keep that as an option?

Reply:
Do you have example figures for that 1 cycle that I pointed you to @mjhossen ?